ci/build.ps1

```diff
@@ -94,6 +94,28 @@ npm.cmd install -g neovim
 Get-Command -CommandType Application neovim-node-host.cmd
 npm.cmd link neovim
 
+#npm.cmd install -g tree-sitter-cli
+#npm.cmd link tree-sitter-cli
+
+mkdir c:\treesitter
+$env:TREE_SITTER_DIR = "c:\treesitter"
+#$env:PATH = "c:\treesitter;$env:PATH"
+$client = new-object System.Net.WebClient
+cd c:\treesitter
+
+if ($bits -eq 32) {
+  $client.DownloadFile("https://github.com/tree-sitter/tree-sitter/releases/download/0.15.5/tree-sitter-windows-x86.gz","c:\treesitter\tree-sitter-cli.gz")
+}
+elseif ($bits -eq 64) {
+  $client.DownloadFile("https://github.com/tree-sitter/tree-sitter/releases/download/0.15.5/tree-sitter-windows-x64.gz","c:\treesitter\tree-sitter-cli.gz")
+}
+python -c "import gzip, shutil; f1,f2 = gzip.open('tree-sitter-cli.gz', 'rb'),  open('tree-sitter.exe', 'wb'); shutil.copyfileobj(f1, f2); f2.close()"
```

Comment:
this is python 2, might as well use `python3` here since python 2 is EOL and non-CI environments are more likely to have python3 (though ideally we would use powershell instead of python).

Reply:
No, there is exactly nothing about this that "is python2" in any way. This just uses whatever python interpreter that is first in PATH and should work with a wide range of python2 and python3 versions. The point of "python 2 is EOL" that "python" refers to python3, and not that it is somehow forbidden to ever refer to the python interpreter as "python" again. This was a mistake made internally in vim only, and does not apply to shell commands.

Reply:
`python` is always python 2 in build.ps1 (see line 83). But I realize now that wouldn't apply for local builds anyway (and powershell would be the ideal approach anyways).

Reply:
> python is always python 2 in build.ps1 (see line 83).

Then it is this part that needs to be fixed to prepare for a python3 only future. The code here just assumes that any python is available as "python".

I tried to look for a decompression API in powershell, but I didn't find any that seemed to be "standard" and worked with short code like this. Feel free to suggest/write the right code :)